modules/storage/variables.tf

```diff
@@ -38,17 +38,23 @@ variable "location" {
 variable "storage_class" {
   description = "The storage class of the storage bucket."
   type        = string
-  default     = "MULTI_REGIONAL"
+  default     = null
```

Comment:
IMHO from a devex perspective we should either explicitly default to `STANDARD` or else say in the description ( null == `STANDARD`)

Reply:
Which one do you prefer?

Reply:
Based on what I see in GKE I think, it makes sense to set default = STANDARD

For example: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/2cb4fae57cb580daba6dc64c7564dcaa7df4efd6/variables.tf#L350
default is 110 but we dont set it to null and instead just set it to 110.

Reply:
Done.